internal/integrationtests/e2e_integration_test.sh

```diff
@@ -13,7 +13,7 @@ echo "====> ✅Step 1: finish preparation"
 echo $DIVIDER
 echo "====> 👀Step 2: build sdp-api, anchor-platform and tss"
 docker-compose -f docker-compose-e2e-tests.yml up --build -d
-sleep 10
+sleep 20
```

Comment:
Given that Kafka takes some time to startup, e2e tests fail intermittently because the tests run before a full startup.